recipes/deepsearch-glm/recipe.yaml

```diff
@@ -0,0 +1,78 @@
+context:
+  name: deepsearch-glm
+  version: 0.26.1
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  url: https://pypi.org/packages/source/${{ name[0] }}/${{ name }}/deepsearch_glm-${{ version }}.tar.gz
+  sha256: c2938e99c4f9f48a8686d3c357778645ec76a78781c89d955720ef78502da830
+  patches:
+    - fix-utfcpp.patch
+
+build:
+  number: 0
+  skip: win
+  script:
+    content: python -m pip install . -vv --no-deps --no-build-isolation
+    env:
+      USE_SYSTEM_DEPS: "on"
+
+requirements:
+  build:
+    - if: build_platform != target_platform
+      then:
+        - python
+        - cross-python_${{ target_platform }}
+
+    - ${{ compiler('cxx') }}
+    - ${{ compiler('c') }}
+    - ${{ stdlib("c") }}
+    - cmake
+    - ${{ "make" if unix else "ninja" }}
+  host:
+    - python
+    - poetry-core
+    - pybind11 >=2.13.1
+    - pip
+    - fmt
+    - cxxopts
+    - nlohmann_json
+    - loguru-cpp
+    - utfcpp
+    - fasttext-for-deepsearch-glm
+    - json_schema_validator
+    - pcre2
+    - sentencepiece
+    - pkg-config
+    - zlib
```

Comment:
I'd much prefer some sort of grouping here, e.g.
```
  host:
    # build tools
    - python
    - poetry-core
    - pybind11 >=2.13.1
    - pip
    - pkg-config
    # libraries
    - cxxopts
    - fmt
    - ...
```
and sort the libraries alphabetically. Also, if you're using `sentencepiece` only as a library from C/C++, you probably want to use `libsentencepiece`.

Reply:
good idea. will do.